pages/osx/networkquality.md

```diff
@@ -0,0 +1,19 @@
+# networkQuality
+
+> Measure the network quality by connecting to the Internet.
+
+- Test the network quality for the default interface:
```

Comment:
Do this command run a speed test always? :thinking: I think the "quality" indicator is the `Responsiveness` (RPM) indicator.

Reply:
I think in my opinion it always runs a speed test.